project-generated-code-backend/Backend/Repository/DatabaseSql/RoomRenovationDatabaseSql.cs

```diff
@@ -0,0 +1,61 @@
+﻿using HealthClinicBackend.Backend.Model.Schedule;
+using HealthClinicBackend.Backend.Repository.Generic;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace HealthClinicBackend.Backend.Repository.DatabaseSql
+{
+    public class RoomRenovationDatabaseSql : GenericDatabaseSql<RoomRenovation>, IRoomRenovationRepository
+    {
+        public RoomRenovationDatabaseSql() : base()
+        {
+        }
+
+        public RoomRenovationDatabaseSql(HealthCareSystemDbContext dbContext) : base(dbContext)
+        {
+        }
+
+        public override List<RoomRenovation> GetAll()
+        {
+            return DbContext.RoomRenovations.ToList();
+        }
+
+        public override RoomRenovation GetById(string id)
+        {
+            return DbContext.RoomRenovations.Find(id);
+        }
+
+
+        public override void Save(RoomRenovation newEntity)
+        {
+            DbContext.RoomRenovations.Add(newEntity);
+            DbContext.SaveChanges();
+        }
+
+        public override void Update(RoomRenovation updateEntity)
+        {
+            DbContext.RoomRenovations.Update(updateEntity);
+            DbContext.SaveChanges();
+        }
+
+        public override void Delete(string id)
+        {
+            var roomRenovation = GetById(id);
+            if (roomRenovation == null) return;
```

Comment:
I think this could be inline if you use ?? return after getting the entity, but I'm not completely sure

Reply:
I will check check this also.

Reply:
I checked this. This is impossible because the operator ?? is classic if statement if returned value is null, then you set the value on the right side. We cannot do the return here. This will stay as it is.
https://stackoverflow.com/questions/52952861/null-coalescing-operator-in-return-statements-c-sharp

Reply:
Yeah, I'm used to Kotlin so I expect too much from c# but this is completely fine too 😄